lib/ts/axios.ts

```diff
@@ -188,6 +189,18 @@ export function responseInterceptor(axiosInstance: any) {
             const preRequestLSS = await getLocalSessionState(false);
             await saveTokensFromHeaders(response);
 
+            const frontToken = response.headers["front-token"];
```

Comment:
Can this be undefined? If so, it may be safer to also expect & check for that in `updateClockSkewUsingFrontToken`